prepare_data/src/main/groovy/org/orbisgis/osm/FormattingForAbstractModel.groovy

```diff
@@ -137,3 +177,65 @@ static int getNbLevels (def row) {
     return result
 }
 
+// define the value of the column width according to the value of width from OSM
```

Comment:
To document the method, use javadoc comment `/**` instead of `//`

Reply:
Done